browser/model/jbds.js

```diff
@@ -7,6 +7,8 @@ let ipcRenderer = require('electron').ipcRenderer;
 let Glob = require("glob").Glob;
 let replace = require("replace-in-file");
 
+var pjson = JSON.parse(fs.readFileSync('./package.json'));
```

Comment:
Could I define a global variable in gulpfile.js and just use that in the browser/model/*.js files and tests?